### PR TITLE
PEP8 fix for models

### PR DESCRIPTION
**Pull request by souravsingh**
I have made an attempt for fixing scripts. Please let me know if I have done any mistakes.

### CODE REVIEW

File: gensim/models/doc2vec.py [hunk 1]
```diff
@@ -59,16 +60,29 @@
...
     from gensim.models.doc2vec_inner import train_document_dbow, train_document_dm, train_document_dm_concat
-    from gensim.models.word2vec_inner import FAST_VERSION  # blas-adaptation shared from word2vec
+    # blas-adaptation shared from word2vec
+    from gensim.models.word2vec_inner import FAST_VERSION
     logger.debug('Fast version of {0} is being used'.format(__name__))
 except ImportError:
     logger.warning('Slow version of {0} is being used'.format(__name__))
-    # failed... fall back to plain numpy (20-80x slower training than the above)
+    # failed... fall back to plain numpy (20-80x slower training than the
+    # above)
```
Comment (gojomo) [inline gensim/models/doc2vec.py]: Are we really strictly enforcing 72-79 char max line lengths? (That's one of the most-often waived odler-standards, given the prevalence of larger/wider/laptop-aspect-ratio screens.)

File: gensim/models/doc2vec.py [hunk 2]
```diff
@@ -59,16 +60,29 @@
...
+            alpha,
+            work=None,
+            train_words=False,
+            learn_doctags=True,
+            learn_words=True,
+            learn_hidden=True,
+            word_vectors=None,
+            word_locks=None,
+            doctag_vectors=None,
+            doctag_locks=None):
```
Comment (gojomo) [inline gensim/models/doc2vec.py]: Is this explosion of parameters, one-to-a-line, really in PEP8? (I didn't notice it on a quick review, and I find it makes the code less readable by taking up so much vertical-space/scrolling between function-begin and function-logic.)

File: gensim/models/doc2vec.py [hunk 3]
```diff
@@ -142,16 +173,33 @@ def train_document_dm(model, doc_words, doctag_indexes, alpha, work=None, neu1=N
                        model.vocab[w].sample_int > model.random.rand() * 2**32]
 
         for pos, word in enumerate(word_vocabs):
-            reduced_window = model.random.randint(model.window)  # `b` in the original doc2vec code
+            reduced_window = model.random.randint(
+                model.window)  # `b` in the original doc2vec code
```
Comment (gojomo) [inline gensim/models/doc2vec.py]: This seems a good example of where strict 79-char line-lengths impair readability. (I'd almost rather lose the comment than move a tiny atomic parameter to a new line!)

File: gensim/models/doc2vec.py [hunk 4]
```diff
@@ -142,16 +173,33 @@ def train_document_dm(model, doc_words, doctag_indexes, alpha, work=None, neu1=N
...
+                word_vocabs[
+                    start:(
+                        pos +
+                        model.window +
+                        1 -
+                        reduced_window)],
+                start)
+            word2_indexes = [
+                word2.index for pos2,
+                word2 in window_pos if pos2 != pos]
```
Comment (gojomo) [inline gensim/models/doc2vec.py]: This split seems absurd to me: a line-break in the MIDDLE of a tuple-desctructuring?!? Wasn't the original fairly idiomatic as a 1-liner (even though slightly over 79 chars)?

**Comment by tmylk** [comment]
This and other PRs closed as needs to be completely re-done with the right coala rules.
Discussed with @souravsingh on gitter

**Pull request closed without merging**